common/src/chain/transaction/transaction_v1.rs

```diff
@@ -85,6 +85,16 @@ impl TransactionV1 {
         }
         Ok(())
     }
+
+    pub fn is_coinbase(&self) -> bool {
```

Comment:
Coinbase doesn't exist anymore. We put the block reward now in the header.

Reply:
Removed coinbase-related checks.